### ISSUE TITLE
SML Helper is not being detected

### ISSUE THREAD

**Issue by OzaKahuu** [LeeTwentyThree/SubnauticaModManager#7]
It detects the zip and i press the button to install it in the game and it just says:

0 Installed
0 updated
0 Failed Installs

I have done all I could think of to fix this problem

**Comment by LeeTwentyThree** [comment]
What mod zip files are you using? ”0 installed” and “0 failed” will occur if you try to install a mod that is not a BepInEx plugin- which means anything made before the living large update

**Comment by LeeTwentyThree** [comment]
This is the latest version of SMLHelper that you are trying to download, correct? A log file would help a lot

**Comment by OzaKahuu** [comment]
> This is the latest version of SMLHelper that you are trying to download, correct? A log file would help a lot

The thing is I tried all the SMLhelper versions I know of, and they all have the same glitch

this log file?

[LogOutput.log](https://github.com/LeeTwentyThree/SubnauticaModManager/files/10472862/LogOutput.log)

**Comment by OzaKahuu** [comment]
Okay I found the issue by accident so apparently my Qmods folder was storing mods that I used in vortex so when I started the game through steam all the old Qmod mods before Living Large were being loaded along with the new mods so i think the Qmod SMLhelper was clashing with the new one and prevented it's detection or whatever so I cleared the folder of the old mods and now all the new mods are working

TLDR: The Old Qmod folder is the culprit 
Solution: Deleted the folder contents
